pkgs/sagemath-environment/tox.ini

```diff
@@ -85,14 +85,15 @@ package_env = .pkg-sagepython
 basepython = {env:SAGE_VENV}/bin/python3
 package_env = .pkg-sagepython-sagewheels-nopypi
 
-[testenv:sagepython-sagewheels-nopypi-norequirements]
-basepython = {env:SAGE_VENV}/bin/python3
-package_env = .pkg-sagepython-sagewheels-nopypi
-
 [testenv:sagepython-sagewheels]
 basepython = {env:SAGE_VENV}/bin/python
 package_env = .pkg-sagepython
 
 [testenv:sagepython-norequirements]
 basepython = {env:SAGE_VENV}/bin/python3
 package_env = .pkg-sagepython
+
+
+[testenv:sagepython-sagewheels-nopypi-norequirements]
```

Comment:
single empty line

Reply:
This break is intended - this sets apart the default environment

Reply:
OK.